### PR TITLE
prevent changing how "Show value as" / "Show achievement levels" from…

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
… misaligning when "Grouped" is chosen
a
b
c
d

### CODE REVIEW

File: webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html [hunk 1]
```diff
@@ -40,7 +40,8 @@ <h2 class="h3 gray-darkest">{{'aggregate-reports.form.heading' | translate}}</h2
       ]">
       <button class="btn btn-success btn-sm green ml-sm mt-xs"
               [disabled]="submissionSubscription"
-              (click)="onGenerateButtonClick()">{{'aggregate-reports.form.submit' | translate}}</button>
+              (click)="onGenerateButtonClick()">{{'aggregate-reports.form.submit' | translate}}
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html]: really hard to tell what changed in these files - can you point it out?

File: webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html [hunk 2]
```diff
@@ -568,24 +573,27 @@ <h3>{{'aggregate-reports.form.section.preview.heading' | translate}} </h3>
...
-                <label class="mr-xs">{{'common.performance-level-display-type-input-label' | translate}}</label>
-                <sb-radio-button-group [options]="options.performanceLevelDisplayTypes"
-                                       [(ngModel)]="settings.performanceLevelDisplayType"
-                                       buttonGroupStyles="btn-group-xs"
-                                       buttonStyles="btn-default"
-                                       analyticsCategory="AggregateQueryBuilder"
-                                       analyticsEvent="Change"></sb-radio-button-group>
-              </span>
+              <div class="pull-right flex-child">
+                <span>
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html]: introduced a `<div class="pull-right flex-child">` that contains a `<span>` and a `<span class="ml-xs">`. Before this had no such `div` and contained two `<span class="pull-right flex-child ml-xs">`

File: webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.html [hunk 3]
```diff
@@ -73,24 +81,26 @@ <h3 class="h3 label-group green pull-left">
...
-              <span class="pull-right flex-child ml-xs">
-                <label class="mr-xs">{{'common.performance-level-display-type-input-label' | translate}}</label>
-                <sb-radio-button-group [options]="displayOptions.performanceLevelDisplayTypes"
-                                       [(ngModel)]="table.performanceLevelDisplayType"
-                                       buttonGroupStyles="btn-group-xs"
-                                       buttonStyles="btn-default"
-                                       analyticsCategory="AggregateReportControl"
-                                       analyticsEvent="Change"></sb-radio-button-group>
-              </span>
+              <div class="pull-right flex-child">
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/aggregate-report/results/aggregate-report.component.html]: introduced a <div class="pull-right flex-child"> that contains a <span> and a <span class="ml-xs">. Before this had no such div and contained two <span class="pull-right flex-child ml-xs">

File: webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html (hunk 2 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html]: hm. can we try:
```html
<div class="flex-children">
  <div class="flex-child grow">column order form control</div>
  <div class="flex-child">performance level form control</div>
  <div class="flex-child">value form control</div>
</div>
```

Comment (wtritch) [reply webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html]: Careful with display:grid.  IE11 needs special prefixed properties and explicit column/row membership

**Comment by aaronsleeper** [comment]
Please add an autoprefixer to your build to protect against such things.
See me if you have questions.

Good point. Thanks Will.

**Comment by dsnyder-fairway** [comment]
moved out of sprint

**Pull request closed without merging**